### ISSUE TITLE
Review should be updated to Revision

### ISSUE THREAD

**Issue by rt4914** [oppia/oppia-android#588]
As per new mocks, Topic Review Tab has been updated to Topic Revision Tab,

<img width="363" alt="Screenshot 2020-01-03 at 12 11 29 PM" src="https://user-images.githubusercontent.com/9396084/71710439-3fa89500-2e22-11ea-83c9-a5564e000459.png">

So, all files related to that needs to be updated like package name, file names, test cases names, xml files names, variable names, strings.xml, drawables, etc.

In short refactor `Review` with `Revision`.

**Comment by rt4914** [comment]
This issue has been assigned to @maheshpaliwal

**Comment by rt4914** [comment]
Unassigning @maheshpaliwal as it was taking too long.

**Comment by rt4914** [comment]
Assigning this to @tanhengyeow

**Comment by rt4914** [comment]
Unassigning @tanhengyeow because of inactivity.

**Comment by Scarlet0103** [comment]
@rt4914 I would like  to work on this....

**Comment by rt4914** [comment]
> @rt4914 I would like to work on this....



@scarlet0103 Thanks for showing interest in Oppia.

Checkout our [wiki](https://github.com/oppia/oppia-android/wiki) first. This will help you in setting up the project and once you sign the CLA and fill the survey form we can assign you issues to work on.
